doc/API.md

```diff
@@ -9,6 +9,7 @@ Note that all addresses are bech32-encoded *version 0* native Segwit `scriptPubK
 | ----------------------------------------- | ---------------------------------------------------- |
 | [`getinfo`](#getinfo)                     | Display general information                          |
 | [`listvaults`](#listvaults)               | Display a paginated list of vaults                   |
+| [`getonchaintxs`](#getonchaintxs)         | Retrieve the vault on chain transactions             |
```

Comment:
Actually, since we don't have a call to fetch all the transactions it could be a `getvaulttxs [vault_outpoint]` and the fact that they are or not onchain is implied by their status in `listvaults`.. Meh